Cargo.toml

```diff
@@ -40,12 +40,12 @@ path = "src/driver.rs"
 clippy_lints = { version = "0.0.170", path = "clippy_lints" }
 # end automatic update
 cargo_metadata = "0.2"
+regex = "0.2"
 
 [dev-dependencies]
 compiletest_rs = "0.3"
 duct = "0.8.2"
 lazy_static = "0.2"
-regex = "0.2"
```

Comment:
rust bootstrap is weird. I have not been able to figure out how else to get non-plugin crates working with UI tests